core/domain/classifier_services.py

```diff
@@ -242,6 +242,51 @@ def get_classifier_training_job_from_model(classifier_training_job_model):
         classifier_training_job_model.data_schema_version)
 
 
+def _get_classifier_training_job_model_by_id(job_id, strict=True):
+    """Gets a classifier training job by a job_id.
+
+    Args:
+        job_id: str. ID of the classifier training job.
+        strict: bool. Raises Exception if ClassifierTrainingJobModel with
+            id not found and strict is set to True. Logs an error, otherwise.
+    
+    Returns:
+        classifier_training_job: ClassifierTrainingJobModel. NDB instance
+            of classifier training job.
+    
+    Raises:
+        Exception: Entity for class ClassifierTrainingJobModel with id not
```

Comment:
Exception: There is no ClassifierTrainingJobModel with the given ID.

Reply:
Done.